modules/onlp/module/auto/onlp.yml

```diff
@@ -261,7 +263,6 @@ definitions:
         - RESET_STATE :    ( 1 << ONLP_SFP_CONTROL_RESET_STATE )
         - RX_LOS :         ( 1 << ONLP_SFP_CONTROL_RX_LOS )
         - TX_FAULT :       ( 1 << ONLP_SFP_CONTROL_TX_FAULT )
-        - TX_DISABLE :     ( 1 << ONLP_SFP_CONTROL_TX_DISABLE )
```

Comment:
We should keep this status bit. It retains the same meaning for SFP, and for QSFP if all 4 lanes are disabled (normal mode).